backend/user-service/routes/user-route.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+
+const userController = require("../controllers/user-controller.js");
+
+router.post("/register", userController.registerUser);
+router.post("/login", userController.loginUser);
+router.post("/delete", userController.deleteUser);
+
```

Comment:
We need more endpoints! Some necessary endpoints including:
1. GET /users/:id
3. UPDATE /users/:id
4. DELETE /users/:id

I am also confused why do you decide to use POST for delete operation here? 
Also, if you already have register and login endpoints in user-service, we might not need to create another auth service. But the complexity for user-service will increase.

Reply:
The reason why I chose to use POST for endpoints eventhough in this case, using DELETE is more semantical, is because of its flexibility and since this is the starting phase of the project, I want to keep the design as exclusive, flexible as possible. However, I guess I'll change this for semantic purpose in the next change.

Reply:
Resolved